NAMESPACE

```diff
@@ -29,7 +29,7 @@ import(ComplexHeatmap)
 import(GAMBLR.helpers)
 import(GAMBLR.viz)
 import(GenomicRanges, except = c("merge", "shift", "union", "intersect", "setdiff", "reduce"))
```

Comment:
`GenomicRnages` should also have the same functions not imported. See [this](https://github.com/morinlab/GAMBLR.utils/issues/6#issuecomment-1688770006) comment.

Reply:
Done.